secure-ios-app/mobile-services.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "1.0",
```

Comment:
I think the latest aerogear-ios-sdk expects this to be a integer now rather than a String. The app crashed for me using the latest SDK with the version defined as a String.